client/utils/gatsby/layout-selector.test.tsx

```diff
@@ -24,8 +34,12 @@ function getComponentNameAndProps(elementType, pathname) {
   shallow.render(<Provider store={store}>{LayoutReactComponent}</Provider>);
   const view = shallow.getRenderOutput();
   return {
-    props: view.props,
+    props: view.props as Record<string, unknown>,
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
     name: view.type.WrappedComponent.displayName
+    // Not really sure what to do with .WrappedComponent here
```

Comment:
This is fine for now.  We want to replace react-test-renderer with react-testing-library and so we can come back to this when we do that.